src/main/java/appeng/util/Platform.java

```diff
@@ -1562,6 +1555,7 @@ public static ItemStack extractItemsByRecipe( final IEnergySource energySrc, fin
 		return ItemStack.EMPTY;
 	}
 
+	// TODO wtf is this?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)